src/api/app/views/webui/user/user_profile_redesign/_info.html.haml

```diff
@@ -0,0 +1,68 @@
+.card.mb-3
+  .card-body
+    .row.mb-3.mb-md-0
+      .col-4.col-sm-2.col-md-12.text-center
+        = image_tag_for(user, size: 100)
```

Comment:
@saraycp whats the reason behind the small image tag? in the previous info box it was 500px

Reply:
just talking about the big viewports

Reply:
Now that you say it... maybe it was a drastic change. So I've updated to 200. You can compare:

Size 100:

![Screenshot_2020-10-19 Profile of Admin(5)](https://user-images.githubusercontent.com/2581944/96461467-b46b8500-1224-11eb-985e-22620465267b.png)

Size 200:

![Screenshot_2020-10-19 Profile of Admin(7)](https://user-images.githubusercontent.com/2581944/96461469-b46b8500-1224-11eb-82c8-6262e56ba096.png)

Reply:
better :+1: